### PR TITLE
New android views cheatsheet

### PR DESCRIPTION
**Pull request by gautamkrishnar**
New android views cheatsheet
- [X] New Instant Answer
  - [X] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**

---

Instant Answer Page: https://duck.co/ia/view/android_views_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @gautamkrishnar

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=android views cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/android_views_cheat_sheet_mobile.png?j1HwwEdrDy)

File: share/goodie/cheat_sheets/json/android-views.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    "EditText (Text field that you can type into)",
+    "Spinner (Click on it to show a list\nof dropdown options Spinner)",
+    "CheckBox (Checkbox with text label)",
+    "Radio button (where you can select one out of a group of radio buttons)",
+    "RatingBar (Star rating)",
+    "Switch (just tap to toggle the state)",
+    "SeekBar (Displays progress and allows you to drag the handle anywhere in the bar)",
+    "SearchView (A search field that you can type a query into)",
+    "ProgressBar (Loading spinner)",
+    "ProgressBar (Horizontal loading indicator)"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/android-views.json]: Lets make the content in brackets shorter and remove the redundant info

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=android views cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/android_views_cheat_sheet_mobile.png?P82chetxCa)

**Comment by marti1125** [comment]
@gautamkrishnar could be a Java Topic?

**Comment by gautamkrishnar** [comment]
@marti1125 good find

File: share/goodie/cheat_sheets/json/android-views.json (hunk 1 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/android-views.json]: Yes that's right

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=android views cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/android_views_cheat_sheet_mobile.png?5rnM35n51c)

**Comment by pjhampton** [comment]
ping @jophab. Is this good to merge?

File: share/goodie/cheat_sheets/json/android-views.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    "sourceName": "GitHub",
+    "sourceUrl": "https://github.com/jaapdejong/android-development-for-beginners--ud837/blob/master/Common%20Android%20Views%20Cheat%20Sheet.pdf"
+  },
+  "aliases": [
+    "android layout",
+    "common android views"
+  ],
+  "template_type": "terminal",
+  "section_order": [
+    "Text View",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/android-views.json]: Is space required here?

File: share/goodie/cheat_sheets/json/android-views.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    "ImageView",
+    "Button",
+    "View",
+    "EditText",
+    "Spinner",
+    "CheckBox",
+    "Radio button",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/android-views.json]: Button

File: share/goodie/cheat_sheets/json/android-views.json [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    "RatingBar",
+    "Switch",
+    "SeekBar",
+    "SearchView",
+    "Loading spinner",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/android-views.json]: Spinner

File: share/goodie/cheat_sheets/json/android-views.json [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    "Horizontal loading indicator"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/android-views.json]: Loading Indicator

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/android-views.json]: Nee superaadaa,,,, 😄

File: share/goodie/cheat_sheets/json/android-views.json [hunk 6]
```diff
@@ -89,12 +89,12 @@
...
       }
     ],
-    "Loading spinner": [
+    "Loading Spinner": [
       {
         "key": "<ProgressBar\nandroid:id=\"@+id/loading_spinner\"\nstyle=\"@style/Widget.AppCompat.ProgressBar\"\nandroid:layout_width=\"wrap_content\"\nandroid:layout_height=\"wrap_content\" />"
       }
     ],
-    "Horizontal loading indicator": [
+    "Horizontal Lading Indicator": [
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/android-views.json]: Correct spelling here

File: share/goodie/cheat_sheets/json/android-views.json [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+        "key": "<Spinner\nandroid:id=\"@+id/sort_by_spinner\"\nandroid:layout_width=\"match_parent\"\nandroid:layout_height=\"wrap_content\" />",
+        "val": "Create SpinnerAdapter in Java code to populate the options"
+      }
+    ],
+    "CheckBox": [
+      {
+        "key": "<CheckBox\nandroid:id=\"@+id/notify_me_checkbox\"\nandroid:layout_width=\"wrap_content\"\nandroid:layout_height=\"wrap_content\"\nandroid:text=\"@string/notify_me\"\nandroid:textAppearance=\"?android:textAppearanceMedium\" />"
+      }
+    ],
+    "Radio button": [
```
Comment (pjhampton) [inline share/goodie/cheat_sheets/json/android-views.json]: This needs to be changed to `Radio Button` with a capital B. The tests will pass then

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/android-views.json]: Thanks for finding @pjhampton 😄

**Comment by daxtheduck** [comment]
@gautamkrishnar this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by pjhampton** [comment]
Awesome! Thanks @gautamkrishnar

**Pull request merged**